React/Views/RCTNavItem.h

```diff
@@ -13,6 +13,9 @@
 
 @interface RCTNavItem : UIView
 
+@property (nonatomic, assign) UINavigationBar *navigationBar;
```

Comment:
These are pointers, they should be strong or possible weak, but definitely not assign.